std/file.d

```diff
@@ -4332,6 +4340,19 @@ private void copyImpl(scope const(char)[] f, scope const(char)[] t,
     assert(readText(t) == "a");
 }
 
+/// issue 19834
```

Comment:
Here is the problem. The three `///` mark this as a public example. You need to use two `//` to turn it into an internal unit test.

```suggestion
// issue 19834
```

Reply:
Oh, thank you very much. I'll try.